library/src/main/java/com/novoda/downloadmanager/Batch.java

```diff
@@ -15,15 +15,15 @@
         this.fileUrls = fileUrls;
     }
 
-    DownloadBatchId getDownloadBatchId() {
+    public DownloadBatchId downloadBatchId() {
```

Comment:
@zegnus would you rather this class is interfaced out like the others and exposed only through a `Creator`? I found it a bit misleading that I could create this class but I can't use or access any information on it except to pass it to the `download-manager`.